templates/build_script.go

```diff
@@ -142,7 +142,7 @@ build_chrome() {
   export PATH="$PATH:$HOME/depot_tools"
   mkdir -p $HOME/chromium
   cd $HOME/chromium
-  fetch --nohooks android --target_os_only=true || true
+  fetch --nohooks android --target_os_only=true --no-history || true
```

Comment:
Nice! - have you verified this works as expected?

Reply:
well, from fetch documentation: `Perform shallow clones, don't fetch the full git history`.
But I am not entirely sure if I did it or not. Cause I was experimenting a lot yesterday and finally built choromium before your fix arrived.

Reply:
Gotcha - I'll kick off a build with this option and see if it works.

Reply:
Looks like at the very least the ordering of flags matter, needs to be: fetch --no-history --nohooks android --target_os_only=true

Still need to verify build works with this setting though.